editor.planx.uk/src/pages/FlowEditor/components/Settings/Submissions/index.tsx

```diff
@@ -0,0 +1,24 @@
+import Box from "@mui/material/Box";
+import Typography from "@mui/material/Typography";
+import React from "react";
+import { FeaturePlaceholder } from "ui/editor/FeaturePlaceholder";
+
+const Submissions: React.FC = () => {
+  return (
+    <>
```

Comment:
nit: It looks like we have an additional level of fragments here which we don't need

Reply:
That's interesting! I directly copy and pasted from an existing tab component and just updated the copy 😅 

I can test it out although I can't think of any reason why we'd need the fragment 🤷

Reply:
It definitely seems like the fragment wrapper was unnecessary so I've removed it and from the associated components here: d67cec818356f245ffa8be7e3bc4f39a09c655cf

Thanks for spotting this @DafyddLlyr, I should have been more observant and nice to add a wee tidy up as part of doing this work 😊

Reply:
Nice outcome to tidy up the other components also 👍